source/making_payments/index.html.md.erb

```diff
@@ -5,120 +5,139 @@ weight: 70
+</style>
 
-You will receive the ``next_url``  to which you should direct the user to
+<br>
+
+<div class="govuk-warning-text">
+  <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+  <strong class="govuk-warning-text__text">
+    <span class="govuk-warning-text__assistive">Warning</span>
+You must not expose the URL with the ``payment_id`` publicly, for example as a URL parameter or in an insecure cookie. You should store it as a secure cookie or in a database.
+  </strong>
+</div>
+
+You will receive the `next_url`  to which you should direct the user to
 complete their payment.
 
-## Tracking the progress of a payment
+## Track the progress of a payment
 
 You can track the progress of a payment while the user is on GOV.UK Pay using
 the <a
 href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/v1/find-payment-by-id"
-target="blank">Find payment by ID</a> call (link opens in new window).
+target="blank">Find payment by ID</a> call [external link]
 
-NOTE: The status of the payment will go through several phases until it either
-succeeds or fails. See the [API reference
-section](/api_reference/#api-reference) for more details.
+The status of the payment will pass through several states until it either
+succeeds or fails. See the [__API reference__
+section](/api_reference/#api-reference) of this documentation for more
+details.
 
-## Choosing the return URL and matching user to payment
+## Choose the return URL and match users to payments
 
 For security reasons, GOV.UK Pay does not add the payment ID or outcome to
-your ``return_url`` as parameters.
+your `return_url` as parameters.
+
+You can match users with payments by using a secure cookie, or creating a
+secure random ID:
+
+### Use a secure cookie
 
-To match up a returning user with their payment, there are two recommended
-methods:
+You can use a secure cookie containing the payment ID from GOV.UK Pay. Your
+service should issue this when the payment is created, before sending the user
+to `next_url`. Users will not be able to decrypt a secure cookie, so a
+malicious user could not alter the payment ID and intercept other users’
+payments.
 
-+ use a secure cookie containing the Payment ID from GOV.UK Pay, issued by
-your service when the payment is created (before sending the user to
-``next_url``). Users will not be able to decrypt a secure cookie, so a
-fraudster could not alter the payment ID and intercept other users’ payments.
+### Create a secure random ID
 
-+ create a secure random ID (such as a UUID) and include this as part of the
-``return_url``, using a different ``return_url`` for each payment. Since a
-securely generated UUID is not guessable, fraudsters will not be able to
-intercept users’ payments.
+You can create a secure random ID,such as a UUID, and include this as part of
+the `return_url`. You should use a different ``return_url`` for each payment.
+Securely generated UUIDs are not guessable, so malicious users will
```

Comment:
Not sure about using the word 'guessable'. Could we just say something like:

> Malicious users will be unable to guess securely generated UUIDs

Is this enough? Is it implied they won't have other means of access using the UUIDs?

Reply:
“Malicious users will be unable to guess securely generated UUIDs” works. The only other way a malicious user could access a secure ID is if it’s not stored securely, so maybe there needs to be an emphasis on that.